pom.xml

```diff
@@ -77,7 +77,9 @@
         <test.exclude.tags></test.exclude.tags>
         <test.include.tags></test.include.tags>
         <cuda.version>cuda11</cuda.version>
-
+        <GPU_ARCHS>ALL</GPU_ARCHS>
+        <CUDF_ENABLE_ARROW_S3>OFF</CUDF_ENABLE_ARROW_S3>
```

Comment:
Do we really want this to ever be `ON`?  This is just to build libcudf so we can link against it, and the libcudf API signatures don't change based on this setting.  IMO this should be hardcoded in the rapids_cpm_find as we do for ARROW_STATIC, etc.

Reply:
Ah, yes, that's better. Updated.